src/servers/src/http/prom_store.rs

```diff
@@ -144,6 +175,42 @@ pub async fn remote_write(
         .into_response())
 }
 
+#[axum_macros::debug_handler]
+#[tracing::instrument(
+    skip_all,
+    fields(protocol = "prometheus", request_type = "remote_write")
+)]
+pub async fn remote_write_without_strict_mode(
+    State(handler): State<PromStoreProtocolHandlerRef>,
+    Query(params): Query<RemoteWriteQuery>,
+    Extension(mut query_ctx): Extension<QueryContextRef>,
+    content_encoding: TypedHeader<headers::ContentEncoding>,
+    RawBody(body): RawBody,
+) -> Result<impl IntoResponse> {
+    let db = params.db.clone().unwrap_or_default();
+    let _timer = crate::metrics::METRIC_HTTP_PROM_STORE_WRITE_ELAPSED
+        .with_label_values(&[db.as_str()])
+        .start_timer();
+
+    let is_zstd = content_encoding.contains(VM_ENCODING);
+    let (request, samples) =
+        decode_remote_write_request_to_row_inserts(is_zstd, body, true).await?;
```

Comment:
method says that it's `without_strict_mode` while the argument is `true`.
Could you please take a look?

Reply:
my mistake, fixed!